.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -1,5 +1,7 @@
 name: 🐞 Bug Report
 description: Report a bug if something isn't working as expected in WooCommerce Core.
+title: "[Bug]: "
+labels: ["type: bug"]
```

Comment:
Not against this change, but only for info the original intent of not adding a label here was that we only wanted it applied to confirmed bugs after teams had been able to reproduce and confirm. For example we get some support requests raised via the bug template (https://github.com/woocommerce/woocommerce/issues?q=is%3Aissue+label%3A%22type%3A+support+request%22+is%3Aclosed).
Can we do a small RFC (if there hasn't already been a discussion about this) also just to check if this would impact any team's existing bug triage process (e.g. in case they exclude those already with the label) or bug reporting metrics.

Reply:
I am no longer convinced about the value of adding a label. I'm ok with having the bug prefix, which is helpful with visually parsing the issues.

Reply:
```suggestion
```